pkgs/misc/ghostscript/default.nix

```diff
@@ -51,6 +51,37 @@ stdenv.mkDerivation rec {
       url = "http://git.ghostscript.com/?p=ghostpdl.git;a=patch;h=d3537a54740d78c5895ec83694a07b3e4f616f61";
       sha256 = "1hr8bpi87bbg1kvv28kflmfh1dhzxw66p9q0ddvbrj72qd86p3kx";
     })
+    ./9.26-CVE-2019-3835-part-1.patch
```

Comment:
So why did you use `fetchpatch` for some patches and local files for other patches?

Reply:
I think a couple of them needed manual adaptation. But this was a long time ago.